memory/migrate_pages.py

```diff
@@ -50,16 +50,23 @@ def setUp(self):
             self.cancel('Test requires two numa nodes to run.'
                         'Node list with memory: %s' % nodes)
 
-        dist = distro.detect()
+        self.dist = distro.detect()
         pkgs = ['gcc', 'make']
-        if dist.name in ["Ubuntu", 'debian']:
+        if self.dist.name in ["Ubuntu", 'debian']:
             pkgs.extend(['libpthread-stubs0-dev',
                          'libnuma-dev', 'libhugetlbfs-dev'])
-        elif dist.name in ["centos", "rhel", "fedora"]:
-            pkgs.extend(['numactl-devel', 'libhugetlbfs-devel'])
-        elif dist.name == "SuSE":
+        elif self.dist.name in ["centos", "rhel", "fedora"]:
+            pkgs.extend(['numactl-devel'])
+            if (self.dist.name == 'rhel' and self.dist.version >= '9'):
+                self.log.info("hugepage tests will not be run, due to\
+                               unavailability of libhugetlbfs packages in\
+                               RHEL-9 or later.")
```

Comment:
@kalpanashtty I think we need to `cancel` the test instead of just printing message if it is not supported.

Reply:
Earlier we were thinking of this i.e canceling the tests if the distro is RHEL-9 or later.  But later based on Geetika and @sacsant  suggestion we took care of handling huge-page tests to be canceled and run THP tests alone. That's the reason we have not to cancel the tests here.

Reply:
then this message is not required here. any way test will get cancelled and the same message can be added where we are cancelling the test

Reply:
This is just a log message with additional info, I thought no harm in giving additional data while it tries to install the packages and later handle canceling part. Is it hard that we need to remove this log info? or no harm in adding the info message?

Reply:
@kalpanashtty nothing harm in having extra info but to go standard of coding it is kind of unnecessary line of code. Thats it, nothing more than that.

Reply:
yeah, this message was added as an information only to let the user know that part of the test is not applicable for RHEL 9. Initial patch cancelled the entire test.